<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuyTicketDialog</class>
 <widget class="QDialog" name="BuyTicketDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>942</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(239,243,245);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>581</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;border-radius: 10px;</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:14pt;</string>
    </property>
    <property name="text">
     <string>2019/10/1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:14pt;</string>
    </property>
    <property name="text">
     <string>出发</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:14pt;</string>
    </property>
    <property name="text">
     <string>D27</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:12pt;</string>
    </property>
    <property name="text">
     <string>北京南</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:18pt;</string>
    </property>
    <property name="text">
     <string>19:10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:12pt;</string>
    </property>
    <property name="text">
     <string>上海</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:18pt;</string>
    </property>
    <property name="text">
     <string>20:10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:14pt;</string>
    </property>
    <property name="text">
     <string>订单总额</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,105,19);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:20pt;</string>
    </property>
    <property name="text">
     <string>200</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:14pt;</string>
    </property>
    <property name="text">
     <string>元</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:18pt;</string>
    </property>
    <property name="text">
     <string>————</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="buy_radio1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:12pt;</string>
    </property>
    <property name="text">
     <string>硬座</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="buy_radio2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:12pt;</string>
    </property>
    <property name="text">
     <string>硬卧</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="buy_radio3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:12pt;</string>
    </property>
    <property name="text">
     <string>软卧</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:16pt;</string>
    </property>
    <property name="text">
     <string>火车票信息</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>581</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;border-radius: 10px;</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:16pt;</string>
    </property>
    <property name="text">
     <string>乘客信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:12pt;</string>
    </property>
    <property name="text">
     <string>姓名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:12pt;</string>
    </property>
    <property name="text">
     <string>身份证号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:12pt;</string>
    </property>
    <property name="text">
     <string>李小明</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:12pt;</string>
    </property>
    <property name="text">
     <string>120111111111111111</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:12pt;</string>
    </property>
    <property name="text">
     <string>联系方式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:12pt;</string>
    </property>
    <property name="text">
     <string>15122222222</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="buy_button">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>870</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: none;
    color: white;
    font-family:Arial;
    padding: 5px 24px;
    text-align: center;
    text-decoration: none;
    font-size: 20px;
    margin: 4px 2px;
background-color: rgb(0,83,135);
font-family:&quot;微软雅黑&quot;, Times, serif;
border-radius: 4px;
}

QPushButton:enabled:hover{
        background-color: rgb(40,102,173);
}
QPushButton:enabled:pressed{
        background-color: rgb(40,102,173);
}</string>
   </property>
   <property name="text">
    <string>预 定</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>680</y>
     <width>581</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;border-radius: 10px;</string>
   </property>
   <widget class="QRadioButton" name="B">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>41</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:12pt;</string>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="D">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:12pt;</string>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="C">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>41</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:12pt;</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="A">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>41</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:12pt;</string>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:16pt;</string>
    </property>
    <property name="text">
     <string>免费选座</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(64,50,38,0.7);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:12pt;</string>
    </property>
    <property name="text">
     <string>靠窗</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(64,50,38,0.7);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:12pt;</string>
    </property>
    <property name="text">
     <string>靠窗</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(64,50,38,0.7);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:12pt;</string>
    </property>
    <property name="text">
     <string>过道</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="F">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(64,50,38);
font-family:&quot;微软雅黑&quot;, Times, serif;
font-size:12pt;</string>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
